e2e/infinite-scroll.e2e-spec.ts

```diff
@@ -8,31 +8,39 @@ import {
 } from 'protractor';
 
 describe('Infinite Scroll', () => {
```

Comment:
FYI - I completely re-worked the visual test to be a little more friendly. Also, the "back-to-top" button required a full-screen approach, because it docks to the window.